.circleci/config.yml

```diff
@@ -30,15 +30,13 @@ jobs:
                   conda create -n testenv --yes pip python=3.8
                   source activate testenv
                   conda install --yes scipy numpy matplotlib
-                  pip install mne pooch tqdm psutil mpi4py joblib nibabel
+                  pip install pooch tqdm mpi4py
```

Comment:
@jasmainak @ntolley 
Do we need `pooch` and `tqdm`? I couldn't find where they were being used currently so they might be some legacy dependency?

Reply:
`pooch` and `tqdm` are actually optional dependencies of MNE-Python but they are needed for the [example](https://jonescompneurolab.github.io/hnn-core/stable/auto_examples/workflows/plot_simulate_somato.html#sphx-glr-auto-examples-workflows-plot-simulate-somato-py) to run even if not explicitly imported.